understanding/20/error-identification.html

```diff
@@ -72,20 +68,21 @@ <h2>Intent of Error Identification</h2>
          that user agents or assistive technologies can use to identify an error and provide
          error information to the user. For example, certain technologies can specify that
          the user's input must not fall outside a specific range, or that a form field is required.
-         Currently, few technologies support this kind of programmatic information, but the
-         Success Criterion does not require, nor prevent it.
-         
+         Currently, few technologies support this kind of programmatic information, but this
+         Success Criterion does not require, nor prevent it. However, other Success Criteria may apply such as <a href="name=role-value">4.1.2 Name, Role, Value</a>, and <a href="info-and-relationships">1.3.1 Info and Relationships</a>.
```

Comment:
This change suggests to me that 4.1.2 or 1.3.1 might require `aria-valid`, `aria-required`, `min`, `max`, or `pattern` might be required. I don't think they do, or even that it could. Those attributes didn't exist when the SC was written. It couldn't possibly have required them.

```suggestion
         Success Criterion does not require, nor prevent it.
```

Reply:
I'd say that's for 1.3.1 and 4.1.2 to then clarify at their end. The statement that they *may* apply though is correct, I'd say.